### PR TITLE
Display searched string as title

### PR DESCRIPTION
**Pull request by StanislavPrime**

### CODE REVIEW

File: examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart [hunk 1]
```diff
@@ -31,7 +30,8 @@ class PuppiesAppBar extends StatelessWidget implements PreferredSizeWidget {
   ) =>
       AppBarTextField(
         searchContainerColor: Colors.blue,
-        title: Text(item.type.asTitle()),
+        // title: Text(item.type.asTitle()),
+        title: _createTitle(context),
```
Comment (StanevPrime) [inline examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart:34]: @StanislavPrime , try to stick to the convention of building stuff. In this case, the better name for  `_createTitle` would be `_buildTitle`

File: examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart [hunk 2]
```diff
@@ -59,6 +59,17 @@ class PuppiesAppBar extends StatelessWidget implements PreferredSizeWidget {
             .add(PuppyListFilterEvent(query: query)),
       );
 
+  Widget _createTitle(BuildContext context) {
+    final lastQuery = context.read<PuppyListBloc>().lastSearchedQuery;
+    return lastQuery != ''
```
Comment (StanevPrime) [inline examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart:64]: @StanislavPrime, check out this method: [isNotEmpty](https://api.dart.dev/stable/2.3.2/dart-core/String/isNotEmpty.html). By using it, you would improve the readability

**Review by StanevPrime — CHANGES_REQUESTED** [review]

File: examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart (hunk 2 above)
Comment (StanislavPrime) [reply examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart]: I changed `lastQuery != ''`  to `lastQuert.isNotEmpty`

File: examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart (hunk 1 above)
Comment (StanislavPrime) [reply examples/favorites_advanced/bloc_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart]: I changed the name of the method  `_createTitle` to `_buildTitle`

**Pull request merged**